src/com/lftechnology/java_training/jan5/ContentInArraysComparator.java

```diff
@@ -0,0 +1,106 @@
+package com.lftechnology.java_training.jan5;
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/**
+ * Compare if two arrays have the same content (doesn't matter if they aren't in order)
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class ContentInArraysComparator {
```

Comment:
ContentInArraysComparator : Is such a bulky name necessary? How about ArrayComparator and whatever we compare with in two arrays are only content related.